test/services/AppealServiceSpec.scala

```diff
@@ -226,6 +228,11 @@ class AppealServiceSpec extends SpecBase {
         result shouldBe true
       }
 
+      "call the audit method in audit service after appeal is submitted successfully (in the above submitAppeal)" in new Setup{
+          verify(mockAuditService, times(1)).audit(ArgumentMatchers.any[JsonAuditModel])(ArgumentMatchers.any[HeaderCarrier],
```

Comment:
Could you not add this to the above test? And just extend the test description to be something like:
`parse the session keys into a model and return true when the connector call is successful and audit the response` as it seems a bit fragile to put this test in without doing any calls.

Reply:
audit() returns 'Unit' not sure how the result against the audit call can be validated with true.

Reply:
So if you remove this test, move the verify call to the test above that. It's all done in one test?

Reply:
Yes sure, Done (Apologies, misinterpreted your initial review!)